server/api.js

```diff
@@ -17,7 +17,7 @@
 const Validator = require('express-validator')
```

Comment:
I know that you're already planning to do this, but just so I don't forget too, you can add tests for these new functions.